capitalizedecorator.rb

```diff
@@ -0,0 +1,7 @@
+require './basedecator'
+
+class CapitalizeDecorator < BaseDecator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] Kindly use the `capitalize` method instead of `upcase` as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=that%20capitalizes%20the%20output%20of%20%40nameable.correct_name). `upcase` will make each character of the word capital, but, `capitalize` will make only the first character of the word capital.

Reply:
Thank you.